### PR TITLE
User management exercise

### PR DESCRIPTION
**Pull request by MattSurabian**
To Do:
- [x] Write exercise copy
- [X] Write exercise instructions
- [X] Write learning objectives
- [X] Implement exercise
- [X] Implement solution

### CODE REVIEW

File: exercises/user-management/solution/user-management.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  become: yes
+  become_method: sudo
+  tasks:
+  # There are several valid ways to do this, and the below is not the most efficient.
+  # However we believe it to be the simplest to arrive at given the boilerplate and
+  # sets us up well for a refactor in the next exercise using loops and variables
+    - name: ensure workshop user is on the box
+      user:
+        state: present
+        name: "workshop"
```
Comment (tkellen) [inline exercises/user-management/solution/user-management.yml]: quotes not needed here

File: exercises/user-management/solution/user-management.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        shell: /bin/bash
+        groups: sudo
+
+    - name: ensure the workshop user's public key is present in authorized_keys
+      authorized_key:
+        user: "workshop"
```
Comment (tkellen) [inline exercises/user-management/solution/user-management.yml]: no quotes needed here (same in a bunch of other places)

File: exercises/user-management/user-management.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+- hosts: all
```
Comment (tkellen) [inline exercises/user-management/user-management.yml]: for consistency, can we call this playbook.yml?

Comment (MattSurabian) [inline exercises/user-management/user-management.yml]: :+1:

**Pull request merged**